Automattic/wp-calypso
# Site Setup:  Support community themes

Closes the  #97248

## Proposed Changes
* Map the theme_type param community to "dot-org"

## Why are these changes being made?
* The current theme selection uses "community" as dot-org theme, and the flow is expecting "dot-org." This mismatching is causing flow to crash and blocking the user to continue. 

## Testing Instructions

* Open an anonymous tab. 
* Go to [calypso live url]/themes)
* Search and select the `astro` theme
* Create a user or log in on WordPress.com
* Select a domain.
* Check if the plan page is not displaying the plans Free, Personal and Premium
* Select the Business plan.
* Check if you can finish the flow.
* Check if you can see a screen asking for install the astra theme



## Pre-merge Checklist
